### PR TITLE
Enforce column ordering

### PR DESCRIPTION
**Pull request by pivnicek**
Unfortunately the test passes even without the extra enforcement, but this was checked against the originating issue #32 to confirm required result.

### CODE REVIEW

File: src/Keboola/DbExtractor/Extractor/PgSQL.php [hunk 1]
```diff
@@ -354,6 +354,8 @@ function ($table) {
                 $tableDefs[$curTable]['columns'][$column['ordinal_position'] - 1]['foreignKeyRef'] =
                     $column['constraint_name'];
             }
+            // make sure columns are sorted by index which is ordinal_position - 1
+            ksort($tableDefs[$curTable]['columns']);
```
Comment (ondrejhlavacek) [inline src/Keboola/DbExtractor/Extractor/PgSQL.php]: `ORDER BY ordinal_position` in the query is self explaining, but I am not sure after https://keboola.slack.com/archives/C02C3GZUW/p1538378583000100 if I am allowed to make such comment :-)

Comment (pivnicek) [reply src/Keboola/DbExtractor/Extractor/PgSQL.php]: oooh :)  I somehow missed that convo!.  unfortunately adding `order by` to the query hurts performance and there is already an issue with performance #30

Comment (ondrejhlavacek) [reply src/Keboola/DbExtractor/Extractor/PgSQL.php]: sure!

Comment (ondrejhlavacek) [reply src/Keboola/DbExtractor/Extractor/PgSQL.php]: i remember the performance issue, but not sure if it was another extractor :-)

**Pull request merged**